jdi-light/src/main/java/com/epam/jdi/light/logger/JDILogger.java

```diff
@@ -47,19 +51,22 @@ public JDILogger(Class clazz) {
     public LogLevels getLogLevel() {
         return logLevel.get();
     }
+
     public void setLogLevel(LogLevels level) {
         logLevel = new Safe<>(level);
         setRootLevel(getLog4j2Level(level));
         setLevel(name, getLog4j2Level(level));
     }
-    private Safe<Integer> logOffDeepness = new Safe<>(0);
+
+    private final Safe<Integer> logOffDeepness = new Safe<>(0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)